src/libs/API.js

```diff
@@ -702,13 +703,38 @@ function CreateIOUSplit(parameters) {
     return Network.post(commandName, parameters);
 }
 
+/**
+ * @returns {Promise}
+ */
+function Wallet_GetOnfidoSDKToken() {
+    const platform = getPlatform();
+    return Network.post('Wallet_GetOnfidoSDKToken', {
+        // We need to pass this so we can request a token with the correct referrer
+        isViaExpensifyCashNative: platform === CONST.PLATFORM.IOS || platform === CONST.PLATFORM.ANDROID,
```

Comment:
Just FYI, I am not a fan of this and I am going to be moving this to follow our cross-platform patterns.